iib/web/migrations/versions/7573241a5156_rename_bundle_deprecation_association_.py

```diff
@@ -0,0 +1,70 @@
+"""Rename bundle deprecation association for merge index.
+
+Revision ID: 7573241a5156
+Revises: eec630370e68
+Create Date: 2021-02-09 13:50:01.905796
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '7573241a5156'
+down_revision = 'eec630370e68'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
```

Comment:
If you're just renaming the table, can you just use [rename_table](https://alembic.sqlalchemy.org/en/latest/ops.html#alembic.operations.Operations.rename_table) ?